modules/learn/pages/clusters-and-availability/hard-failover.adoc

```diff
@@ -21,6 +21,29 @@ Hard failover can be initiated by means of:
 
 For further information on initiating hard failover, see xref:manage:manage-nodes/failover-hard.adoc[Perform Hard Failover].
 
+[#safe-and-unsafe]
+== Hard-Failover Modes: Safe and Unsafe
+
+Either of the following two _modes_ can be specified for hard failover:
+
+* _Safe_ (the default mode).
+When hard failover has been initiated, it is continued by Couchbase Server only in the following circumstances:
+
+** Data-loss cannot result.
+
+** A majority of nodes can still be contacted, following a node’s becoming unresponsive.
+
+** A majority of groups can still be contacted, following a group’s becoming unresponsive.
```

Comment:
Not true. The only condition that determines whether a hard failover is unsafe is if a majority of nodes cannot be contacted to get agreement that the nodes being failed over should be failed over.

Reply:
I have modified the passage, and updated other pages accordingly.  Hopefully, this is closer.